external/Catch2/catch2/catch.hpp

```diff
@@ -10819,7 +10819,7 @@ namespace Catch {
 
     // 32kb for the alternate stack seems to be sufficient. However, this value
     // is experimentally determined, so that's not guaranteed.
-    static constexpr std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ;
+    static constexpr std::size_t sigStackSize = 32768;
```

Comment:
Fix to test failing:
`error: call to non-‘constexpr’ function ‘long int sysconf(int)’`